resource-usage-collect-addon/pkg/addon/agent/agent.go

```diff
@@ -162,18 +158,26 @@ func newAgentController(
 
 func (c *agentController) sync(ctx context.Context, syncCtx factory.SyncContext) error {
 	score := NewScore(c.nodeInformer, c.podInformer)
-	cpuScore, memScore, err := score.calculateScore()
+	cpuScore, memScore, gpuScore, tpuScore, err := score.calculateScore()
 	if err != nil {
 		return err
 	}
 	items := []apiv1alpha2.AddOnPlacementScoreItem{
 		{
 			Name:  "cpuAvailable",
-			Value: int32(cpuScore),
+			Value: cpuScore,
 		},
 		{
 			Name:  "memAvailable",
-			Value: int32(memScore),
+			Value: memScore,
+		},
+		{
+			Name:  "gpuAvailable",
+			Value: gpuScore,
+		},
+		{
+			Name:  "tpuAvailable",
+			Value: tpuScore,
 		},
```

Comment:
Hmm, I'm thinking should we keep the current xxxAvailable as cluster level score and add a new score based on max avalible node.

Reply:
Or at least explain the score is based on cluster max avaliable node.

Reply:
Yes I should add an explanation. 
For an extra score, I think we can do that too, like for example for each cluster, we have `cpuClusterAvailable`, `cpuNodeAvaliable`, `gpuClusterAvaliable`, `gpuNodeAvaliable`, etc.